manifest/kubernetes/ubuntu/ncp-ubuntu.yaml

```diff
@@ -970,10 +970,6 @@ spec:
           # we move the usr kmod files to this dir temporarily before
           # installing new OVS kmod and/or backing up existing OVS kmod backup
             mountPath: /tmp/nsx_usr_ovs_kmod_backup
-
-          # mount linux headers for compiling OVS kmod
-          - name: host-usr-src
-            mountPath: /usr/src
 {{else}}
           # Uncomment these mounts if installing NSX-OVS kernel module
```

Comment:
we can remove code line #L973 - 984.
{{else}}
..
#mountPath: /tmp/nsx_user_ovs_kmod_backup